pkgs/development/ocaml-modules/color/default.nix

```diff
@@ -1,30 +1,36 @@
 {
   lib,
-  fetchurl,
+  fetchFromGitHub,
   buildDunePackage,
   gg,
 }:
 
 buildDunePackage rec {
   pname = "color";
-  version = "0.2.0";
+  version = "0.3.0";
 
   useDune2 = true;
```

Comment:
```suggestion
```

Don’t forget to remove this line.

Reply:
Thank you for pointing out, I have fixed it.